docs/support/faq/i-cannot-login.md

```diff
@@ -33,3 +33,68 @@ others:
 
 You need to contact us to get your account unlocked. Our email address is
 [servicedesk@csc.fi](mailto:servicedesk@csc.fi).
+
+## Why is my SSH client saying "Corrupted MAC on input"?
+
+There is a known issue of **Windows OpenSSH clients**, which are using the LibreSSL library for
+cryptography. The bug happens when the combination of cipher `aes128-ctr` and either the
+`umac-128-etm@openssh.com` or `umac-128@openssh.com` MAC algorithm is used.
+The client will show an error saying `Corrupted MAC on input`.
+
+Here are some links to relevant bug reports:
+
+- [https://github.com/libressl/portable/issues/603](https://github.com/libressl/portable/issues/603)
+- [https://github.com/PowerShell/Win32-OpenSSH/issues/1359](https://github.com/PowerShell/Win32-OpenSSH/issues/1359)
+- [https://github.com/PowerShell/Win32-OpenSSH/issues/2078](https://github.com/PowerShell/Win32-OpenSSH/issues/2078)
+
+If you encounter this issue while trying to log in via SSH, you can try to add an explicit choice
+of MAC algorithm to your SSH client, instead of using the automatically chosen algorithms.
+This is achieved with the `-m <MAC algorithm>` option.
+To find out what MAC algorithms your client technically supports, you can run the command
```

Comment:
which command?

Reply:
The `ssh` command. Maybe unclear, but I'm referring to the "SSH client" in the previous sentence.

Reply:
```suggestion
To find out what MAC algorithms your client technically supports, you can run the `ssh` command.
```